pytorch/inference/docker/1.10/py3/Dockerfile.graviton.e3.cpu

```diff
@@ -45,22 +46,25 @@ RUN keytool -importkeystore -srckeystore /etc/ssl/certs/java/cacerts -destkeysto
     mv /etc/ssl/certs/java/cacerts.jks /etc/ssl/certs/java/cacerts; \
     /var/lib/dpkg/info/ca-certificates-java.postinst configure;
 
-RUN curl -L -o ~/miniconda.sh https://repo.anaconda.com/miniconda/Miniconda3-py38_4.10.3-Linux-aarch64.sh \
- && chmod +x ~/miniconda.sh \
- && ~/miniconda.sh -b -p /opt/conda \
- && rm ~/miniconda.sh
+RUN curl -L -o ~/mambaforge.sh https://github.com/conda-forge/miniforge/releases/download/${MAMBA_VERSION}/Mambaforge-${MAMBA_VERSION}-Linux-aarch64.sh \
```

Comment:
not sure if there is a 1:1 mamba veresion to conda version mapping - cc @tejaschumbalkar - is this an OK to increase this version?

Reply:
There is no mamba to conda version mapping. Mamba just uses conda under the hood, but has faster package resolution. We actually end up still use the `conda` executable for installation in our Dockerfiles since there is a problem using `mamba` due to some way we configure our images. Miniforge by default installs from conda-forge and includes the `mamba` executable as well as `conda`. Miniconda (what was being used before) installs from repo.anaconda and only has `conda`.

Reply:
Should be fine to use the latest version of mamba